ui/src/overridden/backoffice/Importer/ImporterTaskDetails/ReportDetails.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Item, Icon, Table, Message } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+
+const displayRecordLink = (id, label) => (
+  <Link to={id} target="_blank">
+    {label}
+  </Link>
+);
+
+const displayValue = (value, urlGenerator, getId, getTitle) => {
+  return _isEmpty(value)
+    ? ''
+    : displayRecordLink(urlGenerator(getId(value)), getTitle(value));
+};
+
+const displayValues = (values, urlGenerator, getId, getTitle) => {
+  return _isEmpty(values)
+    ? ''
+    : values.map((value, index) => (
+        <span key={index}>
+          {index > 0 ? ', ' : ''}
+          {displayValue(value, urlGenerator, getId, getTitle)}
+        </span>
+      ));
+};
+
+export class ReportDetails extends React.Component {
+  renderErrorMessage = message => {
+    let title;
+    let content;
+    const split = message.split(': ');
+    if (split.length > 1 && split[0] === 'LossyConversion') {
+      title = 'Lossy conversion detected';
+      content = `The literature was not imported because conversion rules for some of the fields were not available. This concerns the following fields: ${split[1]}`;
+    } else {
+      title = 'Something went wrong while processing this document';
+      content = `An unexpected error was encountered: ${message}`;
+    }
+    return (
+      <Message negative>
+        <Message.Header>{title}</Message.Header>
+        <p>{content}</p>
+      </Message>
+    );
+  };
+
+  render() {
+    const { item } = this.props;
+    return (
+      <Item>
+        <Item.Content>
+          {item.success ? (
+            <Table definition compact striped textAlign="center">
```

Comment:
there is a table component which is more compact in terms of code and could be used for this